src/jdk.hotspot.agent/doc/index.html

```diff
@@ -10,220 +10,55 @@ <h1>Using HotSpot Serviceability Agent (SA)</h1>
-
-<tr>
-<td>
-<a name="dumpsysprops"></a>
-dumpsyspropsproc.sh,
-dumpsyspropsproc64.sh,
-dumpsyspropswindbg.bat
-dumpsyspropswindbg64.bat
-</td>
-<td>
-This prints name and value of Java level System properties.
-</td>
-</tr>
-
-<tr>
-<td>
-<a name="heapdump"></a>
-heapdumpproc.sh,
-heapdumpproc64.sh,
-heapdumpwindbg.bat
-heapdumpwindbg64.bat
-</td>
-<td>
-Dumps heap in a file in hprof binary format.
-</td>
-</tr>
-
-<tr>
-<td>
-<a name="heapsum"></a>
-heapsumproc.sh,
-heapsumproc64.sh,
-heapsumwindbg.bat
-heapsumwindbg64.bat
-</td>
-<td>
-Prints summary information on Java heap.
-</td>
-</tr>
-
-
-<tr>
-<td>
-jcoreproc.sh,
-jcoreproc64.sh,
-jcorewindbg.bat
-jcorewindbg64.bat
-</td>
-<td>
-This can retrieve .class files from the debuggee.
-set the environment variable <b>JCORE_PACKAGES</b> to comman separated list of
-packages whose classes have to be retrieved from the core file.
-</td>
-</tr>
-
-<tr>
-<tr>
-<td>
-jstackproc.sh,
-jstackproc64.sh,
-jstackwindbg.bat
-jstackwindbg64.bat
-</td>
-<td>
-used to get java stack trace for all java threads.
-</td>
-</tr>
-
-<tr>
-<td>
-jhistoproc.sh,
-jhistoproc64.sh,
-jhistowindbg.bat
-jhistowindbg64.bat
-</td>
-<td>
-used to get object histogram of java heap.
-</td>
-</tr>
-
-<tr>
-<td>
-permstatproc.sh,
-permstatproc64.sh,
-permstatwindbg.bat
-permstatwindbg64.bat
-</td>
-<td>
-To gather statistics on perm. generation.
-</td>
-</tr>
-
-<a name="mixed_pstack"></a>
-<tr>
-<tr>
-<td>
-pstackproc.sh,
-pstackproc64.sh,
-pstackwindbg.bat
-pstackwindbg64.bat
-</td>
-<td>
-This is cross platform mixed mode pstack utility. This works on any (non-java as well) process, core dump. For java process and core dumps, this prints both java and C/C++ frames. 
-</td>
-</tr>
-
-<tr>
-<td>
-pmapproc.sh,
-pmapproc64.sh,
-pmapwindbg.bat
-pmapwindbg64.bat
-</td>
-<td>
-This is cross platform Solaris pmap-like utility. 
-</td>
-</tr>
-
-<td>
-start-debug-server-proc.sh,
-start-debug-server-proc64.sh,
-start-debug-server-windbg.bat,
-start-debug-server-windbg64.bat,
-start-rmiregistry.bat,
-start-rmiregistry64.bat,
-start-rmiregistry.sh
-start-rmiregistry64.sh
-</td>
-<td>
-These scripts are used to run SA remotely. 
-</td>
-</tr>
-</table>
-
-<h3>C2 Compilation Replay</h3>
+<h3>Compilation Replay</h3>
 <p>
-When a java process crashes in compiled method, usually a core file is saved.
-The C2 replay function can reproduce the compiling process in the core.
-<a href="c2replay.html">c2replay.html</a>
+When a java process crashes in a compiled method, usually a core file is saved.
+The compiler replay function can reproduce the compiling process in the core.
+See <a href="cireplay.html">cireplay.html</a>
 
 <h3>Debugging transported core dumps</h3>
 <p>
 When a core dump is moved from the machine where it was produced to a
-difference machine, it may not always be possible for SA to debug the same.
-More info. on debugging on transported core dumps is in
+difference machine, it may not always be possible for SA to debug it.
```

Comment:
"difference machine" rather than "different machine " is not your typo but if you had time while we are here... 8-)

Reply:
Done.